### ISSUE TITLE
5.9.20210221.0: test suite is failing

### ISSUE THREAD

**Issue by kloczek** [universal-ctags/ctags#2894]
As I'm now able to build ctags after fixing #2885 I found that test suite is failing.
```
[tkloczko@barrel ctags-p5.9.20210221.0]$ make check
/usr/bin/gcc -std=gnu99 -Wall   -I/usr/include/libxml2     -O2 -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fdata-sections -ffunction-sections -flto=auto -flto-partition=none  -Wl,-z,relro -Wl,--as-needed -Wl,--gc-sections -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -flto=auto -flto-partition=none -fuse-linker-plugin -o ctags  main/ctags-cmd.o libctags.a -lxml2  -ljansson  -lyaml  -lseccomp
\
if test -n "${ZSH_VERSION+set}"; then set -o SH_WORD_SPLIT; fi; \
if test x = x1; then            \
        VALGRIND=--with-valgrind;       \
fi;                                     \
if ! test x = x; then   \
        SHOW_DIFF_OUTPUT=--show-diff-output;            \
fi;                                                     \
builddir=$(pwd); \
if ! test xpython3 = x; then    \
        PROG=python3;           \
        SCRIPT=./misc/units.py; \
        if type cygpath > /dev/null 2>&1; then  \
                builddir=$(cygpath -m "$(pwd)");        \
                if ! test x/bin/sh = x; then    \
                        SHELL_OPT=--shell=$(cygpath -m /bin/sh);        \
                fi;     \
        else    \
                if ! test x/bin/sh = x; then    \
                        SHELL_OPT=--shell=/bin/sh;      \
                fi;     \
        fi;     \
else    \
        PROG=/bin/sh;           \
        SCRIPT=./misc/units;    \
fi;     \
mkdir -p ${builddir}/Tmain && \
\
c="${SCRIPT} tmain \
        --ctags=./ctags \
        --units= \
        ${VALGRIND} \
        ${SHELL_OPT} \
        ${SHOW_DIFF_OUTPUT}"; \
TRAVIS= APPVEYOR= CIRCLECI= GITHUBACTIONS=\
        ${PROG} ${c} ./Tmain ${builddir}/Tmain

Testing alias-for-unknown-language
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing allow-null-tag
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing abnormal-output-file-names
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing broken-tagname-in-ectags-format
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing broken-tagname-quiet
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing broken-extradef
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing broken-tagname
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing broken-fielddef
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing case-insensitive-pattern
------------------------------------------------------------
skipped (feature "case-insensitive-filenames" is not available in /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/./ctags)

Testing broken-langdef
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing c-anon-counter
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing client-vista-vim-fields-expectation
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing combination-of-fields-Zs
------------------------------------------------------------
stdout                                                      passed

Testing combination-of-fields-zkK
------------------------------------------------------------
stdout                                                      passed

Testing clear-aliases
------------------------------------------------------------
stdout                                                      passed
exit                                                        passed

Testing dot-ctags-with-indentation
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing dynamic-kinds
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing disable-fixed-field
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing cxx-how-kinds-defs-are-shared-or-copyed
------------------------------------------------------------
stdout                                                      passed

Testing emacs-modeline-downcased-mode
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing emacs-modeline-non-alnum-in-head
------------------------------------------------------------
stdout                                                      passed

Testing emacs-modeline-non-alnum-in-head2
------------------------------------------------------------
stdout                                                      passed

Testing emacs-modeline-non-alnum-in-tail
------------------------------------------------------------
stdout                                                      passed

Testing emacs-modeline-uppercased-marker
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing disable-languages
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing enable-kind-postfix
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing enable-kind-postfix-with-wildcard
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing e-ctags-output
------------------------------------------------------------
stdout                                                      passed

Testing enable-kind-prefix
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing enable-non-existing-kind
------------------------------------------------------------
stderr                                                      passed

Testing enable-kind-prefix-with-wildcard
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing excmd-combine-backward
------------------------------------------------------------
stdout                                                      passed

Testing excmd-combine
------------------------------------------------------------
stdout                                                      passed

Testing extension-vs-pattern
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing extras-field-for-pseudo-tags
------------------------------------------------------------
stdout                                                      passed

Testing extras-field
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing epoch-field
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing errors-about-parser-own-extras
------------------------------------------------------------
stderr                                                      passed

Testing filter-option-write-to-file
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing filter-option
------------------------------------------------------------
stdout                                                      passed

Testing fixed-field-handling-in-json-format
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing extras-long
------------------------------------------------------------
stdout                                                      passed

Testing force-initializing-option
------------------------------------------------------------
stderr                                                      passed

Testing gcov-sandbox
------------------------------------------------------------
skipped (feature "gcov" is not available in /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/./ctags)

Testing generate-anon-ids
------------------------------------------------------------
stdout                                                      passed

Testing input-encoding-option
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed
tags                                                        passed

Testing interactive-notice-output
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing interactive-mode
------------------------------------------------------------
stdout                                                      passed

Testing interactive-option-write-to-file
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing invalid-encoding-option
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing interactive-resource-management
------------------------------------------------------------
stdout                                                      passed

Testing json-limit-recursion
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing json-output-boolean-fields
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing json-output-for-broken-input
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing errors-in-options-roles
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing json-output-much-parser-fields
------------------------------------------------------------
exit                                                        passed

Testing json-output-format
------------------------------------------------------------
stdout                                                      passed

Testing json-output-to-file
------------------------------------------------------------
stdout                                                      passed

Testing fixed-field-handling
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing kind-long-lregex
------------------------------------------------------------
stdout                                                      passed

Testing json-ptag-in-list-extras
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing kind-abnormal-spec
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing kind-long
------------------------------------------------------------
stdout                                                      passed

Testing kind-and-role-defaults
------------------------------------------------------------
stdout                                                      passed

Testing langmap-option
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing langmap-option-including-patterns
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing language-names
------------------------------------------------------------
exit                                                        passed

Testing languages-and-language-force-options
------------------------------------------------------------
stderr                                                      passed
exit                                                        passed

Testing list-aliases
------------------------------------------------------------
stdout                                                      passed

Testing list-excludes
------------------------------------------------------------
stdout                                                      passed

Testing list-fields-with-prefix
------------------------------------------------------------
stdout                                                      passed

Testing list-extras
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing list-kinddef-flags
------------------------------------------------------------
stdout                                                      passed

Testing list-kinds-full
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing list-language-fields
------------------------------------------------------------
stdout                                                      passed

Testing list-map-extensions
------------------------------------------------------------
stdout                                                      passed

Testing kinddef
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing list-mline-regex-flags
------------------------------------------------------------
stdout                                                      passed

Testing list-map-patterns
------------------------------------------------------------
stdout                                                      passed

Testing list-mtable-regex-flags
------------------------------------------------------------
stdout                                                      passed

Testing list-pseudo-tags
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing list-regex-flags
------------------------------------------------------------
stdout                                                      passed

Testing list-params
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing list-roles-with-kind-names
------------------------------------------------------------
stdout                                                      passed

Testing list-fields
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing load-conf-files-under-cwd-no-dot
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing list-subparsers-all
------------------------------------------------------------
stdout                                                      passed

Testing load-conf-files-under-cwd
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing load-conf-files-under-home
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing load-dot-ctags-twice
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing lregex-clear-patterns
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing lregex-kind-letter
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing lregex-list-kinds-full
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing lregex-list-kinds-uniquely
------------------------------------------------------------
stdout                                                      passed

Testing lregex-list-kinds
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing lregex-reject-reserved-kind
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing lregex-kind-name
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing map-for-unknown-language
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing lregex-unexpected-regex-kind
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing map-lang-option
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing maxdepth
------------------------------------------------------------
stdout                                                      passed

Testing map-removing
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing mline-no-advance
------------------------------------------------------------
stderr                                                      passed

Testing mtable-avoid-infinite-loop-at-the-eof
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing mtable-pinning
------------------------------------------------------------
stderr                                                      passed

Testing mtable-mutual-entering
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing multi-roles
------------------------------------------------------------
stdout                                                      passed

Testing mtable-stats
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing nameless-long-option
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing nested-mio
------------------------------------------------------------
stdout                                                      passed
exit                                                        passed

Testing nested-subparsers-multilines
------------------------------------------------------------
stdout                                                      passed

Testing list-roles
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing no-input-encoding-option
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing null-description-in-pseudo-tag
------------------------------------------------------------
stdout                                                      passed
exit                                                        passed

Testing omit-long-patterns-etags
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing omit-long-patterns
------------------------------------------------------------
stdout                                                      passed

Testing option-dump-options
------------------------------------------------------------
stdout                                                      passed

Testing option-dump-keywords
------------------------------------------------------------
stdout                                                      passed

Testing kinds-all-with-spec
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing option-echo-and-force-quit
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing option-exclude-exception
------------------------------------------------------------
stdout                                                      passed

Testing option-exclude-including-fsep
------------------------------------------------------------
stdout                                                      passed

Testing option-exclude-simple
------------------------------------------------------------
stdout                                                      passed

Testing option-options-directory
------------------------------------------------------------
stderr                                                      passed
exit                                                        passed

Testing option-options-maybe
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing option-extradef-in-mtable
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing option-totals-extra
------------------------------------------------------------
stdout                                                      passed

Testing option-extras-enabling-all
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing option-use-slash-as-filename-separator
------------------------------------------------------------
skipped (feature "win32" is not available in /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/./ctags)

Testing nested-subparsers
------------------------------------------------------------
stdout                                                      passed

Testing optlib-dir-option
------------------------------------------------------------
stderr                                                      passed

Testing optlib-message-flag
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing output-encoding-option
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed
tags                                                        passed

Testing option-pseudo-tags
------------------------------------------------------------
stdout                                                      passed

Testing parser-init-useCork
------------------------------------------------------------
stdout                                                      passed

Testing output-format-option
------------------------------------------------------------
stdout                                                      passed

Testing output-input-field-with-no-escape
------------------------------------------------------------
stdout                                                      passed

Testing parser-own-fields-clearing-all
------------------------------------------------------------
stdout                                                      passed

Testing parser-own-fields
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing pretend-option
------------------------------------------------------------
stdout                                                      passed

Testing parser-own-extras
------------------------------------------------------------
stdout                                                      passed

Testing ptag-dont-emit-to-stdout-by-default
------------------------------------------------------------
stdout                                                      passed

Testing pattern-length-limit
------------------------------------------------------------
stdout                                                      passed

Testing ptag-field-sec
------------------------------------------------------------
stdout                                                      passed

Testing ptag-extra-desc
------------------------------------------------------------
stdout                                                      passed

Testing output-file-resource-management
------------------------------------------------------------
stdout                                                      passed

Testing ptag-kind-desc
------------------------------------------------------------
stdout                                                      passed

Testing ptag-in-optlib-parser
------------------------------------------------------------
stdout                                                      passed

Testing ptag-proc-cwd
------------------------------------------------------------
stdout                                                      passed

Testing ptag-role-desc
------------------------------------------------------------
stdout                                                      passed

Testing ptag-output-excmd
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing ptag-kind-sep
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing readtags-broken-input
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing readtags-alias
------------------------------------------------------------
stdout                                                      passed

Testing readtags-combine
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing readtags-error-in-sorter
------------------------------------------------------------
exit                                                        passed

Testing readtags-list-pseudo-tags
------------------------------------------------------------
stdout                                                      passed

Testing readtags-escaping
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing readtags-qualifier-broken-exp
------------------------------------------------------------
stderr                                                      passed
exit                                                        passed

Testing readtags-qualifier-end-field
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing readtags-qualifier-begin
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing readtags-qualifier-op-length
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing readtags-qualifier-op-downcase-upcase
------------------------------------------------------------
stdout                                                      passed

Testing readtags-qualifier-op-regexp-quote
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing readtags-qualifier-op-string-append
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing readtags-simple-listing
------------------------------------------------------------
stdout                                                      passed
exit                                                        passed

Testing readtags-qualifier-regex
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing readtags-qualifier
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing readtags-sorter-if-sform-order
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing readtags-sorter-if-sform
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing regex-for-unknown-language
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing readtags-sorter
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing regex-patterns-from-file
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing sandbox-crash
------------------------------------------------------------
skipped (feature "debug" is not available in /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/./ctags)

Testing run-as-etags
------------------------------------------------------------
stderr                                                      passed
exit                                                        passed

Testing sandbox-unknown-submode
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing sandbox-default-req
------------------------------------------------------------
stdout                                                      failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox-default-req.d/stdout-diff.txt)
stderr                                                      failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox-default-req.d/stderr-diff.txt)
exit                                                        failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox-default-req.d/exit-diff.txt)

Testing sandbox-no-eager-guessing
------------------------------------------------------------
stdout                                                      failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox-no-eager-guessing.d/stdout-diff.txt)
stderr                                                      failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox-no-eager-guessing.d/stderr-diff.txt)
exit                                                        failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox-no-eager-guessing.d/exit-diff.txt)

Testing scopesep
------------------------------------------------------------
stdout                                                      passed

Testing selector-dont-select-disabled-lang
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing sandbox-with-eager-guessing
------------------------------------------------------------
stdout                                                      failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox-with-eager-guessing.d/stdout-diff.txt)
stderr                                                      failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox-with-eager-guessing.d/stderr-diff.txt)
exit                                                        failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox-with-eager-guessing.d/exit-diff.txt)

Testing selector-xml-root-elelement
------------------------------------------------------------
stdout                                                      passed

Testing roledef
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing sorted-list-languages
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing sql-null-tag-warning
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing sorted-help-message
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed

Testing tab-in-parser-specific-field
------------------------------------------------------------
stdout                                                      passed

Testing sandbox
------------------------------------------------------------
stdout                                                      failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox.d/stdout-diff.txt)
stderr                                                      failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox.d/stderr-diff.txt)
exit                                                        failed (diff: /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox.d/exit-diff.txt)

Testing tag-relative-option-no-optarg
------------------------------------------------------------
exit                                                        passed

Testing tag-relative-option-in-etags
------------------------------------------------------------
stdout                                                      passed

Testing tags-output-boolean-fields
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing tmain-example
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed
exit                                                        passed
tags                                                        passed

Testing tmain-skip-example
------------------------------------------------------------
skipped (example: no such feature)

Testing subparser-direction
------------------------------------------------------------
stdout                                                      passed

Testing trace-option
------------------------------------------------------------
skipped (feature "debug" is not available in /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/./ctags)

Testing two-inputs-requiring-scope-stack
------------------------------------------------------------

Testing w32-slash-in-exclude-option
------------------------------------------------------------
skipped (feature "win32" is not available in /home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/./ctags)

Testing warn-reusing-kind-letter
------------------------------------------------------------
stdout                                                      passed

Testing wrong-parser-name-in-field
------------------------------------------------------------
stderr                                                      passed
exit                                                        passed

Testing unreadable-input
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing wildcard-in-lang-of-fields-option
------------------------------------------------------------
stdout                                                      passed

Testing xformat-boolean-fields
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing xref-output-to-file
------------------------------------------------------------
stdout                                                      passed

Testing tag-relative-option
------------------------------------------------------------
stdout                                                      passed

Testing xformat-and-parser-own-field
------------------------------------------------------------
stdout                                                      passed
stderr                                                      passed

Testing xref-ptag-in-list-extras
------------------------------------------------------------
stdout                                                      passed

Testing xformat-common-fields
------------------------------------------------------------
stdout                                                      passed

Failed tests
============================================================
sandbox-default-req/stdout-compare
sandbox-default-req/stderr-compare
sandbox-default-req/exit-compare
sandbox-no-eager-guessing/stdout-compare
sandbox-no-eager-guessing/stderr-compare
sandbox-no-eager-guessing/exit-compare
sandbox-with-eager-guessing/stdout-compare
sandbox-with-eager-guessing/stderr-compare
sandbox-with-eager-guessing/exit-compare
sandbox/stdout-compare
sandbox/stderr-compare
sandbox/exit-compare

make: *** [Makefile:6072: tmain] Error 1
```

**Comment by masatake** [comment]
Could show `/home/tkloczko/rpmbuild/BUILD/ctags-p5.9.20210221.0/Tmain/sandbox.d/*.txt` ?

**Comment by masatake** [comment]
I guess though seccomp library was available at built-time, the runtime environment doesn't support seccomp.

**Comment by masatake** [comment]
Reproduced on Fedora34.

**Comment by masatake** [comment]
I home the issue is fixed in the latest code.

**Comment by kloczek** [comment]
FYI here is version of the patch which I'm cuttently using against 5.9.20210509.0:
```patch
get rid of AX_PROG_CC_FOR_BUILD aclocal macro to fix  #2885

This commit fixes #2885 which is caused by use custom AX_PROG_CC_FOR_BUILD
aclocal macro which does not honor $CC abd other env varuiables.
With that simplification packcc binary is compiled and linked ans non installeable
program with only standard AC_PROG_CC macro.
It simplifirs as well automake by building packcc as noinst_PROGRAMS, and because
that custon clean-local targert can be removed as well.

--- a/configure.ac~     2021-05-01 04:30:25.000000000 +0100
+++ b/configure.ac      2021-05-02 10:56:43.501464938 +0100
@@ -252,59 +252,12 @@
 AC_PROG_RANLIB
 AC_C_BIGENDIAN

-# https://www.gnu.org/software/autoconf-archive/ax_prog_cc_for_build.html
-AX_PROG_CC_FOR_BUILD
-
-# We need to compile and run a program on the build machine.
-AC_MSG_CHECKING(for cc for build)
-if test "$cross_compiling" = "yes"; then
-  CC_FOR_BUILD="${CC_FOR_BUILD-cc}"
-else
-  CC_FOR_BUILD="${CC_FOR_BUILD-$CC}"
-  CFLAGS_FOR_BUILD="${CFLAGS_FOR_BUILD-$CFLAGS}"
-  CPPFLAGS_FOR_BUILD="${CPPFLAGS_FOR_BUILD-$CPPFLAGS}"
-  LDFLAGS_FOR_BUILD="${LDFLAGS_FOR_BUILD-$LDFLAGS}"
-fi
-AC_MSG_RESULT($CC_FOR_BUILD)
-AC_ARG_VAR(CC_FOR_BUILD,[build system C compiler])
-AC_ARG_VAR(CFLAGS_FOR_BUILD,[CFLAGS for build system C compiler])
-AC_ARG_VAR(CPPFLAGS_FOR_BUILD,[CPPFLAGS for build system C compiler])
-AC_ARG_VAR(LDFLAGS_FOR_BUILD,[LDFLAGS for build system C compiler])
-
-AC_ARG_VAR(AR,[ar command or path])
-AC_ARG_VAR(RANLIB,[ranlib command or path])
-AC_ARG_VAR(WINDRES,[windres command or path, used with mingw-w64])
 AC_SUBST([WINDRES])

-if test "x${AR}" = "x" ; then
-    AR=ar
-fi
-
 if test "x${WINDRES}" = "x" ; then
     WINDRES=windres
 fi

-if test "${with_sparse_cgcc}" = "yes"; then
-       REAL_CC="${CC}"
-       CC="cgcc"
-       AC_SUBST([REAL_CC])
-
-       PRETTY_ARG_VAR([CGCC_CFLAGS], [cgcc specific flags],
-                      [-Wsparse-all])
-else
-       CGCC_CFLAGS=""
-fi
-
-if test "${enable_coverage_gcov}" = "yes"; then
-       COVERAGE_CFLAGS="--coverage"
-       COVERAGE_LDFLAGS="--coverage"
-       AC_DEFINE(ENABLE_GCOV)
-fi
-AC_SUBST([COVERAGE_CFLAGS])
-AC_SUBST([COVERAGE_LDFLAGS])
-
-AC_PROG_LN_S
-AC_CHECK_PROG(STRIP, strip, strip, :)
 AC_SYS_LARGEFILE

 AC_CHECK_PROGS([perl_found], [perl], [yes], [no])
--- a//Makefile.am~     2021-05-10 19:34:41.000000000 +0100
+++ b//Makefile.am      2021-05-10 19:37:16.829955370 +0100
@@ -34,30 +34,21 @@
 CLEANFILES =
 MOSTLYCLEANFILES =

-clean-local:
-       rm -f packcc$(BUILD_EXEEXT)
-       @if test "$(top_srcdir)" != "$(top_builddir)"; then \
-               rm -rf $(OPTLIB2C_SRCS); \
-               rm -rf $(TXT2CSTR_SRCS); \
-       fi
-
 bin_PROGRAMS = ctags
 noinst_LIBRARIES = libctags.a

-noinst_PROGRAMS =
+noinst_PROGRAMS = packcc
 noinst_PROGRAMS += mini-geany

 if HAVE_STRNLEN_FOR_BUILD
 EXTRA_CPPFLAGS_FOR_BUILD = -DPCC_USE_SYSTEM_STRNLEN
 endif

-PACKCC = $(top_builddir)/packcc$(BUILD_EXEEXT)
-
 cc4b_verbose = $(cc4b_verbose_@AM_V@)
 cc4b_verbose_ = $(cc4b_verbose_@AM_DEFAULT_V@)
 cc4b_verbose_0 = @echo CC4BUILD "  $@";
-$(PACKCC): $(top_srcdir)/misc/packcc/src/packcc.c
-       $(cc4b_verbose)$(CC_FOR_BUILD) $(CFLAGS_FOR_BUILD) $(CPPFLAGS_FOR_BUILD) $(EXTRA_CPPFLAGS_FOR_BUILD) $(LDFLAGS_FOR_BUILD) -o $@ $(top_srcdir)/misc/packcc/src/packcc.c
+
+packcc_SOURCES = misc/packcc/src/packcc.c

 if USE_READCMD
 bin_PROGRAMS+= readtags
@@ -161,14 +152,13 @@
 $(TXT2CSTR_SRCS): $(TXT2CSTR) Makefile
 endif

-packcc_verbose = $(packcc_verbose_@AM_V@)
-packcc_verbose_ = $(packcc_verbose_@AM_DEFAULT_V@)
-packcc_verbose_0 = @echo PACKCC "    $@";
-SUFFIXES += .peg
-.peg.c:
-       $(packcc_verbose)$(PACKCC) -o $(top_builddir)/peg/$(*F) "$<"
-.peg.h:
-       $(packcc_verbose)$(PACKCC) -o $(top_builddir)/peg/$(*F) "$<"
+.peg.c: packcc$(EXEEXT)
+       $(top_builddir)/packcc$(EXEEXT) -o $(top_builddir)/peg/$(*F) "$<"
+.peg.h: packcc$(EXEEXT)
+       $(top_builddir)/packcc$(EXEEXT) -o $(top_builddir)/peg/$(*F) "$<"
+
+$(PEG_SRCS) $(PEG_HEADS): packcc$(EXEEXT)
+
 # You cannot use $(PACKCC) as a target name here.
 $(PEG_SRCS) $(PEG_HEADS): $(PACKCC) Makefile
 dist_libctags_a_SOURCES = $(ALL_LIB_HEADS) $(ALL_LIB_SRCS)

```
I really don't care about crosscompilation as long as non-cross build without LTO does not work.
